tests/components/deconz/test_binary_sensor.py

```diff
@@ -76,22 +75,13 @@ async def test_binary_sensors(hass):
     data = deepcopy(DECONZ_WEB_REQUEST)
     data["sensors"] = deepcopy(SENSORS)
     gateway = await setup_deconz_integration(hass, get_state_response=data)
```

Comment:
For the future, probably return the config entry instead of the gateway.

Reply:
Will fix in future pr

Reply:
Fixed in https://github.com/home-assistant/core/pull/41362